app/src/main/java/org/mozilla/tv/firefox/ext/Application.kt

```diff
@@ -0,0 +1,18 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.tv.firefox.ext
+
+import android.app.Application
+import android.content.Intent
+import org.mozilla.tv.firefox.MainActivity
+
+/**
+ * Creates a new intent and starts MainActivity. Note that this can be used to bring the app
+ * to the foreground when it is in the background
+ */
+fun Application.startMainActivity() {
```

Comment:
nit: this is unused

Reply:
It felt potentially useful in the future.  🤷‍♂  I can remove it if you'd prefer

Reply:
Removed

Reply:
> It felt potentially useful in the future. 🤷‍♂ I can remove it if you'd prefer

In general, even if it's potentially useful, unused code should be removed because it can break and people won't know about it because it's not being tested regularly. For example, if for some reason we change the intent required to start MainActivity and only then this code gets used, it may not work as expected. Another case is if the platform changes but we don't.